test.js

```diff
@@ -48,4 +48,29 @@ describe('csslint-stylish', () => {
 
     assert(CSSLint.hasFormat('stylish'), 'csslint should be stylish')
   })
+
+  it('should be able to report from CSSLint without undefined line output', () => {
+    // assert(!CSSLint.hasFormat('stylish'), 'csslint should not be stylish')
+
+    CSSLint.addFormatter(reporter)
+
+    const res = CSSLint.verify('.class {\n  color: red\n}\n')
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>Too many errors. (74% scanned).